src/org.freedesktop.PackageKit.xml

```diff
@@ -591,6 +591,46 @@
       </doc:doc>
     </property>
 
+    <!--*********************************************************************-->
+    <method name="GetResults">
+      <doc:doc>
+        <doc:description>
+          <doc:para>
+            If the last offline action was completed successfully, returns a list
+            of packages that were actioned on. Otherwise return an error code and
+            error description.
+          </doc:para>
+        </doc:description>
+      </doc:doc>
+      <arg type="as" name="package_ids" direction="out">
+        <doc:doc>
+          <doc:summary>
+            <doc:para>
+              An array of package IDs.
+            </doc:para>
+          </doc:summary>
+        </doc:doc>
+      </arg>
+      <arg type="s" name="error_code" direction="out">
```

Comment:
This should be `u`, we should return the actual error code here. We should also explicitly state how the "no error" condition looks like if we use an unsigned integer  (which could be returning `PK_ERROR_ENUM_UNKNOWN` or MAXUINT).
TBH, I think in this case the most straightforward thing to do would be to also return a "success" boolean that is false if an error is set, and write in the documentation that error code and description are only valid if success is false :-)